src/server/qgsservercachemanager.cpp

```diff
@@ -139,7 +148,7 @@ bool QgsServerCacheManager::deleteCachedDocuments( const QgsProject *project ) c
 QByteArray QgsServerCacheManager::getCachedImage( const QgsProject *project, const QgsServerRequest &request, QgsAccessControl *accessControl ) const
 {
   bool cache = true;
-  QString key = getCacheKey( cache, accessControl );
+  QString key = getCacheKey( cache, accessControl, "", project, request );
```

Comment:
`QString()` should probably be used instead of `""`.

Reply:
Or `QStringLiteral`...

Reply:
I think that `QString()` is more appropriate in this case.